.github/workflows/focus-ios-l10n-locales.yml

```diff
@@ -0,0 +1,54 @@
+name: L10n Focus iOS screenshots (one set of locales only)
```

Comment:
This file holds the code that is reused and is called from focus-ios-l10n-screenshots.yml